models/project.go

```diff
@@ -305,3 +307,21 @@ func deleteProjectByID(e Engine, id int64) error {
 
 	return updateRepositoryProjectCount(e, p.RepoID)
 }
+
+// LoadRepository loads repository of a given project
+func (p *Project) LoadRepository() error {
+	return p.loadRepository(x)
+}
+
+// loadRepository loads repository of a given project
+func (p *Project) loadRepository(e Engine) error {
+	var repo = Repository{}
+	if p.Type == ProjectTypeRepository {
+		if _, err := e.ID(p.RepoID).Get(&repo); err != nil {
+			log.Info("failed getting repo %w", err)
```

Comment:
%w is a go 1.13 feature

```suggestion
			log.Info("failed getting repo %v", err)
```